app/models/review.rb

```diff
@@ -1,5 +1,7 @@
 class Review < ApplicationRecord
   validates :body, presence: true
+  validates :rating, presence: true
+  validates_numericality_of :rating, greater_than_or_equal_to: 1, less_than_or_equal_to: 5, message: "Rating must be between 1 - 5"
```

Comment:
Review this to set your hound config to not yell about line length: https://docs.google.com/document/d/1Huuzt3gIuU8gom4SScOMtEhXEqHmJ7iPKt5Yo4Rpe60/edit